docs/hooks/hook_civicrm_relativeToAbsolute.md

```diff
@@ -0,0 +1,56 @@
+# hook_civicrm_relativeToAbsolute
+
+## Summary
+
+This hook is called to alter the from and to of relative date.
```

Comment:
This summary doesn't really make sense - not sure what the hook does but this summary doesn't tell me.